skl2onnx/common/_container.py

```diff
@@ -3,10 +3,41 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-
+import inspect
+import re
+import traceback
 import six
+import sys
 from ..proto import helper
 from .interface import ModelContainer
+from ._apply_operation import __dict__ as dict_apply_operation
```

Comment:
a little hacky, why not just import it and dir all items?

Reply:
pylint raises a warning for from … import * so I usually avoid it.